UKHomeOffice/nmsw-ui
# Test NMSW-1280 & NMSW-1275

- Added test for NMSW-1344, the user is taken to task page after FAL5 check is completed
- Added test for NMSW-1275 amg timeout error for fal5 and fal6 upload
- Added test for NMSW-1280 AMG Insufficient data error for FAL5/6 upload

## To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose amg-error-validation.feature
check-your-answers-page.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/amg-error-validation.feature
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/check-your-answers-page.feature
```
